### PR TITLE
Better errors for missing imports

### PR DESCRIPTION
**Pull request by erezsh**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -50,6 +50,7 @@ mysql = ["mysql-connector-python"]
 pgsql = ["psycopg2"]
```
Comment (sirupsen) [inline pyproject.toml:50]: Should we call this extra `postgres` instad? `pgsql` isn't even the name of the package 😅

File: pyproject.toml [hunk 2]
```diff
@@ -50,6 +50,7 @@ mysql = ["mysql-connector-python"]
...
 snowflake = ["snowflake-connector-python"]
 presto = ["presto-python-client"]
+oracle = ["cx_Oracle"]
```
Comment (sirupsen) [inline pyproject.toml:53]: Ah good! What about BigQuery, Redshift, .. any others?

File: data_diff/database.py [hunk 3]
```diff
@@ -22,6 +22,24 @@ def parse_table_name(t):
     return tuple(t.split("."))
 
 
+def import_helper(package: str = None, text=""):
```
Comment (sirupsen) [inline data_diff/database.py:25]: Maybe we should add the extra name to the list of database tables in the README? Or at least in the `How to install` example, list all of them?

File: pyproject.toml (hunk 1 above)
Comment (erezsh) [reply pyproject.toml:50]: I think it was to stay in the same format as mysql/mssql.

Anyway, I'm fine with either one.

Comment (sirupsen) [reply pyproject.toml:50]: Let's switch to `postgres` then

File: data_diff/database.py (hunk 3 above)
Comment (sirupsen) [reply data_diff/database.py:25]: I think it's the same issue: make it easier for people to figure out what drivers to install and when

**Comment by sirupsen** [comment]
Let's get this in for 0.1 too

**Review by sirupsen — APPROVED** [review]

**Pull request merged**